### PR TITLE
PSFR-1927 Add resettlement worker detail to the Overview page

### PR DESCRIPTION
**Pull request by abdulrkalam**

### CODE REVIEW

File: server/routes/prisoner-overview/__snapshots__/prisonerOverviewController.test.ts.snap [hunk 1]
```diff
@@ -5101,6 +5117,10 @@ exports[`prisonerOverview should render the prisoner overview page with correct
                 <th class="govuk-table__header" scope="row">Community Offender Manager</th>
                 <td class="govuk-table__cell govuk-table__cell--min-width-100">John Doe</td>
               </tr>
+              <tr class="govuk-table__row">
+                  <th class="govuk-table__header" scope="row">Resettlement Worker</th>
+                  <td class="govuk-table__cell govuk-table__cell--min-width-100">-</td>
```
Comment (JamesBoobier) [inline server/routes/prisoner-overview/__snapshots__/prisonerOverviewController.test.ts.snap:5122]: Probably worth adding a test for when the resettlement worker is not null

**Review by JamesBoobier — APPROVED** [review]
